[2025-05-20T14:04:49.913+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-20T14:04:44.536674+00:00/task_id=transform_api permission to 509
[2025-05-20T14:04:49.913+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-20T14:04:44.536674+00:00/task_id=transform_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-20T14:04:44.536674+00:00/task_id=transform_api'
[2025-05-20T14:04:49.942+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-20T14:04:44.536674+00:00/task_id=transform_api permission to 509
[2025-05-20T14:04:49.943+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-20T14:04:44.536674+00:00/task_id=transform_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-20T14:04:44.536674+00:00/task_id=transform_api'
[2025-05-20T14:04:49.977+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dimensional_model.transform_api manual__2025-05-20T14:04:44.536674+00:00 [queued]>
[2025-05-20T14:04:49.984+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dimensional_model.transform_api manual__2025-05-20T14:04:44.536674+00:00 [queued]>
[2025-05-20T14:04:49.985+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2025-05-20T14:04:49.996+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): transform_api> on 2025-05-20 14:04:44.536674+00:00
[2025-05-20T14:04:50.001+0000] {standard_task_runner.py:60} INFO - Started process 216 to run task
[2025-05-20T14:04:50.003+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'etl_dimensional_model', 'transform_api', 'manual__2025-05-20T14:04:44.536674+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/etl_load_postgres_dag.py', '--cfg-path', '/tmp/tmpghg4_j55']
[2025-05-20T14:04:50.005+0000] {standard_task_runner.py:88} INFO - Job 5: Subtask transform_api
[2025-05-20T14:04:50.048+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-20T14:04:44.536674+00:00/task_id=transform_api permission to 509
[2025-05-20T14:04:50.048+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-20T14:04:44.536674+00:00/task_id=transform_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-20T14:04:44.536674+00:00/task_id=transform_api'
[2025-05-20T14:04:50.049+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_dimensional_model.transform_api manual__2025-05-20T14:04:44.536674+00:00 [running]> on host 9e8b630445d3
[2025-05-20T14:04:50.252+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_dimensional_model' AIRFLOW_CTX_TASK_ID='transform_api' AIRFLOW_CTX_EXECUTION_DATE='2025-05-20T14:04:44.536674+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-20T14:04:44.536674+00:00'
[2025-05-20T14:04:50.277+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-20T14:04:50.287+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_dimensional_model, task_id=transform_api, execution_date=20250520T140444, start_date=20250520T140449, end_date=20250520T140450
[2025-05-20T14:04:50.335+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-20T14:04:50.357+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
